types/stake.go

```diff
@@ -0,0 +1,66 @@
+package types
+
+import (
+	abci "github.com/tendermint/abci/types"
+	"github.com/tendermint/go-crypto"
+)
+
+// status of a validator
+type BondStatus byte
+
+// nolint
+const (
+	Unbonded  BondStatus = 0x00
+	Unbonding BondStatus = 0x01
+	Bonded    BondStatus = 0x02
+	Revoked   BondStatus = 0x03
+)
+
+// validator for a delegated proof of stake system
+type Validator interface {
+	GetStatus() BondStatus    // status of the validator
+	GetAddress() Address      // owner address to receive/return validators coins
```

Comment:
GetOwner would be clearer (Address could be address of the validator)

Reply:
addressed